src/views/Correction.vue

```diff
@@ -669,10 +669,10 @@ export default class Correction extends Mixins(CommonMixin, DateMixin, EnumMixin
     }
 
     const data: any = {
-      [FilingTypes.CORRECTION]: {
```

Comment:
There is only the one filing type here, so I decided to be explicit instead of generic.

Reply:
Reverted (since the old code was consistent with the other local filings). Resolving.